examples/mnist/train_mnist_with_double_buffering.py

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env python
```

Comment:
Do we need a new file for double buffering? It will double our maintenance cost, so I would like to be conservative.

Reply:
Ok, I will fix it.